src/main/java/com/conveyal/r5/analyst/WebMercatorGridPointSet.java

```diff
@@ -60,10 +60,6 @@ public WebMercatorGridPointSet(int zoom, int west, int north, int width, int hei
         // TODO don't copy, just have a cache that lazy loads and crops from base pointset
         if (base != null) {
             base.linkageCache.asMap().forEach((key, baseLinkage) -> {
-                // TODO handle the case where the new grid is not completely contained by the base grid.
```

Comment:
Although this is no longer a TODO the comment should still be retained, explaining why it's OK to do what we're doing (the cells are unreachable by transit or streets so might as well be unlinked).

Reply:
👍